lib/rmg/ObjectManager.cpp

```diff
@@ -112,6 +112,9 @@ int3 ObjectManager::findPlaceForObject(const rmg::Area & searchArea, rmg::Object
 				continue;
 			
 			obj.setPosition(tile);
+
+			if (obj.getVisibleTop().y < 0)
+				continue;
```

Comment:
Consider moving this condition inside weight functions. Thus you can control which objects should satisfy this condition and which shouldn't

Reply:
All of the objects should fit on a screen.